### PR TITLE
Adding a new option 'none' in outputFormat.

### PR DESCRIPTION
**Pull request by koushdey**
So that the user can choose to not use `-o` in kubectl command

Fixes: [12852](https://github.com/microsoft/azure-pipelines-tasks/issues/12852)

### CODE REVIEW

File: Tasks/KubernetesV1/task.loc.json [hunk 1]
```diff
@@ -410,7 +410,8 @@
       "required": false,
       "options": {
         "json": "json",
-        "yaml": "yaml"
+        "yaml": "yaml",
+        "none": "none"
```
Comment (ds-ms) [inline Tasks/KubernetesV1/task.loc.json:414]: Add one more for `no output` maybe? 
`none` could imply `no outputFormat`

**Comment by koushdey** [comment]
I have already added test cases for the feature. Do you suggest to add test for empty string?

**Comment by OliwiaStrzelec** [comment]
> I have already added test cases for the feature. Do you suggest to add test for empty string?

I can see that development is ongoing here. Is there possibility to pass custom output like `-o custom-columns=":metadata.name`? Now this task forces me to use `yaml/json` which I don't like. I was trying to bypass it and passed empty string, i.e.:
```
- task: Kubernetes@1
    displayName: Get deployments
    inputs:
      connectionType: $(connType)
      kubernetesServiceEndpoint: $(serviceEndpoint)
      arguments: deployments --no-headers -o custom-columns=":metadata.name"
      command: get 
      checkLatest: true
      outputFormat: ''
```
It "kind of" worked, but polluted the output with `commandOutput` and a coma.

**Comment by koushdey** [comment]
@OliwiaStrzelec Working on custom output.
